### DISCUSSION TITLE
Disable supported languages for system fields

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#12867] [Feature Requests]
It is possible to limit the languages for content with the automatically created `languages` collection.

But it is not easily possible to disable all the languages that are available to select in the `Translation Strings` page for example:

![image](https://user-images.githubusercontent.com/10547444/164104745-49993abe-d33b-4063-aedd-09b39f423b14.png)

As you can see, the list is huge.

My use case is, that I am from Germany and I develop my directus apis in `English`, but I need the user interface to be `German` for my customers, who are actually using the cms.

But setting this up is very painful, as I always have to scroll through a long list of available languages for each field that I translate.

This is why I propose to have some sort of dropdown with selectable fields within the settigs page, where I can select, which languages I want to support.

A few related improvements that also would be nice, but aren't the main point of this discussion:
- make it so that directus tells you if certain `collection fields` or `translation strings`are not yet available in every enabled language

Related to #11568 I think, but not sure if its a duplicate

**Comment by JonL1** [comment]
This would be useful. Scrolling and searching through the whole list of languages is time consuming and prone to error.

**Reply by JonL1** [reply]
I am really surprised this is not popular because it's a productivity killer. @rijkvanzanten

**Reply by rijkvanzanten** [reply]
> I am really surprised this is not popular because it's a productivity killer. @rijkvanzanten


Yeah! 🤷🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Reply by JonL1** [reply]
I really can't believe nobody has an issue with this! Because if that is so it means that nobody is using the translation functionality :D

**Reply by rijkvanzanten** [reply]
> I really can't believe nobody has an issue with this! Because if that is so it means that nobody is using the translation functionality :D

Nahh it's really not that black and white :) I think there's many more reasons. An important one is that this feature request didn't even show up on GitHub for a while because of their 1000 discussion limits, and another that people most likely didn't even realize or found this feature request before (I've seen a bunch of duplicates..)

You've probably noticed that we're cleaning out all old feature requests that didn't get any real upvotes. The hope is that we get the list of open feature requests down to a point where people can actually see/explore them all, and easily find the one that talks about the problem they're facing. That way, we prevent a ton of duplicates and feature requests that go stale, even though they're valuable! I'm also hopeful that this particular pain point can be resolved by adding inline-search for dropdown interfaces, so you can just type `Engl` without having to scroll through anything (which would be a great feature request!) alas, I think you catch my drift :D

**Reply by JonL1** [reply]
I was kind of joking of course. Sorry you had to write all that. I feel bad now. The translation function is a clear 80/20 if not more. I was just surprised at the low visibility it got.

How long would you suggest waiting until the big clean up is done and the number of FR is a tad more manageable to create new ones?

**Reply by rijkvanzanten** [reply]
Oh hahah 😅 Little hard to tell in text sometimes! You'd be surprised at how often people are.. not joking 🙃 

> How long would you suggest waiting until the big clean up is done and the number of FR is a tad more manageable to create new ones?

You're welcome to shoot things in now! We're only cleaning up anything older than 90 days. It'll probably be another couple days to be completely done 🙂

**Reply by LasseRosenow** [reply]
So do I understand this correct, that we can create a new issue with the same feature request as one of those that have been closed, if we think it is important?

Because you say some might just not got so many stars because they haven't been seen at all?

**Reply by rijkvanzanten** [reply]
> So do I understand this correct, that we can create a new issue with the same feature request as one of those that have been closed, if we think it is important?
> 
> Because you say some might just not got so many stars because they haven't been seen at all?

That's exactly right. There were 2500 open feature requests, and github only shows up to 1000, so it was literally impossible to see them all / find what you're looking for. By properly categorizing the ones that are already in-works or under-review, and by closing ones that haven't gotten any engagement after 3 months, i'm hopeful we can keep the list of open feature requests small enough to a point where community members actually have a chance to see and explore them 🙂

**Reply by LasseRosenow** [reply]
Aaah I totally get it now haha thanks a lot for the clarification :) ❤️
